celer/dropin_sklearn.py

```diff
@@ -264,6 +263,134 @@ def _more_tags(self):
         return {'multioutput': False}
 
 
+class ElasticNet(Lasso_sklearn):
```

Comment:
I don't think it makes a huge difference but it should inherit from the sklearn ElasticNet class instead